beetmoverscript/templates/firefox_nightly_repacks.yml

```diff
@@ -76,5 +76,12 @@ mapping:
       destinations:
         - {{ upload_date }}-{{ branch }}-l10n/firefox-{{ version }}.{{ locale }}.{{ platform }}.installer.exe
         - latest-{{ branch }}-l10n/firefox-{{ version }}.{{ locale }}.{{ platform }}.installer.exe
-
   {% endif %}
+  {% for partial, from_buildid in partials.iteritems() %}
+    {{ partial }}:
+      s3_key: firefox-{{ branch }}-{{ version }}-{{ platform }}-{{ locale }}-{{ from_buildid }}-{{ buildid }}.partial.mar
+      destinations:
+        - {{ upload_date }}-{{ branch }}/firefox-{{ branch }}-{{ version }}-{{ platform }}-{{ locale }}-{{ from_buildid }}-{{ buildid }}.partial.mar
+        - latest-{{ branch }}/firefox-{{ branch }}-{{ version }}-{{ platform }}-{{ locale }}-{{ from_buildid }}-{{ buildid }}.partial.mar
```

Comment:
I believe we shouldn't publish l10n mars to latest-branch. It's where en-US artifacts live.

Reply:
Good catch, I will update!